tmt/utils.py

```diff
@@ -521,6 +521,7 @@ def run(
 
         # Run the command in an interactive mode if requested
         if interactive:
+            # Interactive mode can return non-zero if the last command failed, ignore errors here.
```

Comment:
Belongs to the commit where `suppress` is added.

Reply:
Thanks, fixed.